tests/local/test_playback.py

```diff
@@ -65,24 +71,58 @@ def setUp(self):  # noqa: N802
     def tearDown(self):  # noqa: N802
         pykka.ActorRegistry.stop_all()
 
+    def assert_state_is(self, state):
+        self.assertEqual(self.playback.get_state().get(), state)
+
+    def assert_current_track_is(self, track):
+        self.assertEqual(self.playback.get_current_track().get(), track)
+
+    def assert_current_track_is_not(self, track):
+        self.assertNotEqual(self.playback.get_current_track().get(), track)
+
+    def assert_current_track_index_is(self, index):
+        tl_track = self.playback.get_current_tl_track().get()
+        self.assertEqual(self.tracklist.index(tl_track).get(), index)
+
+    def assert_next_tl_track_is(self, tl_track):
+        current = self.playback.get_current_tl_track().get()
+        self.assertEqual(self.tracklist.next_track(current).get(), tl_track)
```

Comment:
You can get the same check with one round trip less using new APIs you added :-)

``` python
self.assertEqual(self.tracklist.next_tlid().get(), tl_track.tlid)
```